adafruit_ble_cycling_speed_and_cadence.py

```diff
@@ -126,13 +131,13 @@ class CyclingSpeedAndCadenceService(Service):
         "Chain Ring",
     )
 
-    def __init__(self, service=None):
+    def __init__(self, service: Optional[Service] = None) -> None:
         super().__init__(service=service)
         # Defer creating buffer until we're definitely connected.
         self._measurement_buf = None
 
     @property
-    def measurement_values(self):
+    def measurement_values(self) -> Optional[Type[CSCMeasurementValues]]:
```

Comment:
This doesn't return `None`. It always returns a `CSCMeasurementValues`.
```suggestion
    def measurement_values(self) -> CSCMeasurementValues:
```

Reply:
It looks like if the packet length is 0, it will return `None`; should I still remove the `Optional` (and it looks like `Type`)?

Reply:
Ah, I see, yes, you're right, but why `Type[CSCMeasurementValues]` instead of `CSCMeasurementValues` ?

Reply:
It looks like the actual type of `CSCMeasurementValues` IS a type itself (running the equivalent code in CPython at least), so I think you're right.  I'll remove the `Type`.  Thanks!  New to `namedtuple` so it threw me for a loop!